src/routes/+page.svelte

```
@@ -14,6 +16,22 @@
     const languages = $derived(data.languages);
 
+    const locale = $derived((data.locale ?? "en") as Locale);
+    const ui = $derived(t(locale));
+
+    const familiarityTitlesByLocale: Record<Locale, readonly string[]> = {
+        en: ["Working Knowledge", "Familiarity", "Additional Technologies", "Basic knowledge"],
+        de: ["Grundkenntnisse", "Grundwissen", "Zusätzliche Technologien", "Vertrautheit"]
+    };
+
+    function normalizeTitle(title: string): string {
+        return title.trim().toLocaleLowerCase();
```

ESLint

[error] Missing JSDoc comment.

(jsdoc/require-jsdoc)